zebra-network/src/address_book/tests/prop.rs

```diff
@@ -33,4 +33,19 @@ proptest! {
             prop_assert!(duration_since_last_seen <= MAX_PEER_ACTIVE_FOR_GOSSIP);
         }
     }
+
+    /// Test if peers that are probably unreachable are not listed for a reconnection attempt.
```

Comment:
Nit: remove negation from the docs as well

```suggestion
    /// Test that only peers that are reachable are listed for reconnection attempts.
```